src/DBConnector/Connector.java

```diff
@@ -0,0 +1,205 @@
+package DBConnector;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+ /**
+  * Please read here:
+  * to run this code, you will need to install jdbc connector that connects the java program to
+  * the database. JDBC is the jar file, and you will need to import it to the referenced library.
+  * In order to check if the data has been written in the database, you can either use terminal
+  * or mysqlWorkbench to view the table. But, you will need to connect to the database first by
+  *  using the provided hostname, username, and password (see field variables);
+  */
+
+ /**
+  * Description:
+  * this is the database connector you will need to use to communicate with the databased hosted on
+  * a remote machine. Pass in the IP of the remote machine where the database is hosted, also your
+  * authorized username and password to the database. we do not need to run mySQL server.
+  */
+
+public class Connector {
```

Comment:
A Builder pattern would provide a clean way to instantiate your Connector, and as I've mentioned elsewhere putting your parameters in a .gitignore'd file will protect your secrets (though you'll have to have a way to provide that secret file to your server).
Are you intentionally leaving the class level fields package-private?